remc2/portability/port_sdl_joystick.cpp

```diff
@@ -412,6 +413,10 @@ void gamepad_event_mgr(gamepad_event_t *gpe)
 		flight_mode = 0;
 	}
 
+	if (SDL_JoystickGetButton(m_gameController, gpc.button_options_menu - 1)) {
```

Comment:
when the options/pause menu pops up the navigation mode activates automatically based on a set_scene() call, so there should be no need for these 3 lines here
  if (SDL_JoystickGetButton(m_gameController, gpc.button_options_menu - 1)) { ... }

Reply:
Sure I will change it